Review.md

```diff
@@ -1,6 +1,11 @@
 # Please answer the following questions
 
 1.  In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+`describe()` is used to describe a set of scenarios. usually for grouping `it()`s. `it()` is used to describe a single scenario.
 2.  What is the point of `Test Driven Development`? What do you think about this approach?
+`Test Driven Development` exists to provide a very secure development lifecycle. "confidence. Trust in the code." I think it slows down production time, unless you have 99% of the project specced out in the first place.
 3.  What are `mocks`? What are a good use cases for them?
```

Comment:
Missing an answer here. Reference the training kit first, then search google if you can't find the answer in training kit.